<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TeamGit</string>
  </property>
  <property name="layoutDirection" >
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>57</y>
     <width>825</width>
     <height>605</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QSplitter" name="verticalSplitter1" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection" >
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="tabPosition" >
        <enum>QTabWidget::South</enum>
       </property>
       <property name="tabShape" >
        <enum>QTabWidget::Triangular</enum>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>383</width>
          <height>557</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>&amp;Files</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QComboBox" name="projectsComboBox" />
         </item>
         <item>
          <widget class="QSplitter" name="splitter" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QTreeView" name="projectFilesView" />
           <widget class="QWidget" name="layoutWidget" >
            <layout class="QVBoxLayout" >
             <item>
              <widget class="QTreeView" name="stagedFilesView" >
               <property name="selectionMode" >
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="autoExpandDelay" >
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="commitButton" >
               <property name="text" >
                <string>&amp;Commit Staged</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QTreeView" name="unstagedFilesView" >
            <property name="selectionMode" >
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="autoExpandDelay" >
             <number>0</number>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>383</width>
          <height>557</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>&amp;Branches/tags</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QSplitter" name="splitter_2" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QTreeView" name="branchesView" />
           <widget class="QWidget" name="layoutWidget" >
            <layout class="QVBoxLayout" >
             <item>
              <widget class="QTreeView" name="tagsView" />
             </item>
             <item>
              <widget class="QPushButton" name="newTagButton" >
               <property name="text" >
                <string>Add &amp;tag</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QSplitter" name="horizontalSplitter1" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget" >
        <layout class="QVBoxLayout" >
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="LogMessage" >
             <property name="text" >
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ResetLogButton" >
             <property name="minimumSize" >
              <size>
               <width>120</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text" >
              <string>Reset to full log</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTreeView" name="logView" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="rootIsDecorated" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QSplitter" name="horizontalSplitter2" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QTabWidget" name="commitLogTabs" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>5</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="baseSize" >
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="layoutDirection" >
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="tabShape" >
          <enum>QTabWidget::Triangular</enum>
         </property>
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_3" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>404</width>
            <height>309</height>
           </rect>
          </property>
          <attribute name="title" >
           <string>&amp;Info</string>
          </attribute>
          <layout class="QHBoxLayout" >
           <item>
            <layout class="QVBoxLayout" >
             <item>
              <layout class="QVBoxLayout" >
               <item>
                <layout class="QHBoxLayout" >
                 <item>
                  <widget class="QLabel" name="label" >
                   <property name="text" >
                    <string>Author</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="commit_author" >
                   <property name="readOnly" >
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" >
                 <item>
                  <widget class="QLabel" name="label_2" >
                   <property name="text" >
                    <string>Date</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="commit_date" >
                   <property name="readOnly" >
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QFrame" name="Log" >
               <property name="frameShape" >
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow" >
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QTextEdit" name="commit_log" >
                  <property name="readOnly" >
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>404</width>
            <height>309</height>
           </rect>
          </property>
          <attribute name="title" >
           <string>&amp;Diff</string>
          </attribute>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QTextEdit" name="commit_diff" >
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="action_New" />
   </widget>
   <widget class="QMenu" name="menu_Edit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Pull" />
    <addaction name="action_CherryPick" />
    <addaction name="action_Merge" />
    <addaction name="action_Options" />
   </widget>
   <widget class="QMenu" name="menu_View" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <addaction name="action_Refresh" />
   </widget>
   <widget class="QMenu" name="menu_Connection" >
    <property name="title" >
     <string>&amp;Management</string>
    </property>
    <addaction name="actionAdd_Repositories" />
    <addaction name="actionAdd_View" />
    <addaction name="actionDelete_Repository" />
    <addaction name="actionRemove_View" />
   </widget>
   <widget class="QMenu" name="menu_Tools" >
    <property name="title" >
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionNew_Tag" />
    <addaction name="actionNew_Branch" />
    <addaction name="action_Delete_Branch" />
    <addaction name="action_Remote_branches" />
    <addaction name="actionPush" />
   </widget>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menu_Edit" />
   <addaction name="menu_View" />
   <addaction name="menu_Connection" />
   <addaction name="menu_Tools" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>662</y>
     <width>825</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>27</y>
     <width>825</width>
     <height>30</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="action_New" />
   <addaction name="action_Pull" />
   <addaction name="action_Refresh" />
   <addaction name="separator" />
   <addaction name="action_CherryPick" />
   <addaction name="action_Merge" />
   <addaction name="separator" />
   <addaction name="action_Options" />
   <addaction name="separator" />
   <addaction name="actionNew_Tag" />
   <addaction name="actionNew_Branch" />
   <addaction name="action_Delete_Branch" />
   <addaction name="separator" />
   <addaction name="actionPush" />
  </widget>
  <action name="action_New" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/main/add.png</normaloff>:/main/add.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Pull" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/main/bottom.png</normaloff>:/main/bottom.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Quick Pull</string>
   </property>
  </action>
  <action name="action_Refresh" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/main/reload_page.png</normaloff>:/main/reload_page.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Refresh</string>
   </property>
  </action>
  <action name="action_Options" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/main/configure.png</normaloff>:/main/configure.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Options</string>
   </property>
  </action>
  <action name="action_CherryPick" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/main/today.png</normaloff>:/main/today.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;CherryPick</string>
   </property>
  </action>
  <action name="action_Merge" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/main/bookmark_toolbar.png</normaloff>:/main/bookmark_toolbar.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Merge</string>
   </property>
  </action>
  <action name="action_Cut" >
   <property name="text" >
    <string>&amp;Cut</string>
   </property>
  </action>
  <action name="actionAdd_Repositories" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/main/adept_install.png</normaloff>:/main/adept_install.png</iconset>
   </property>
   <property name="text" >
    <string>Add &amp;Repository</string>
   </property>
  </action>
  <action name="actionAdd_View" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/main/view_fit_window.png</normaloff>:/main/view_fit_window.png</iconset>
   </property>
   <property name="text" >
    <string>Add &amp;View</string>
   </property>
  </action>
  <action name="actionDelete_Repository" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/main/adept_remove.png</normaloff>:/main/adept_remove.png</iconset>
   </property>
   <property name="text" >
    <string>Remove Repository</string>
   </property>
  </action>
  <action name="actionRemove_View" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/main/remove.png</normaloff>:/main/remove.png</iconset>
   </property>
   <property name="text" >
    <string>Remove View</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/main/messagebox_info.png</normaloff>:/main/messagebox_info.png</iconset>
   </property>
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionNew_Tag" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/main/rss_tag.png</normaloff>:/main/rss_tag.png</iconset>
   </property>
   <property name="text" >
    <string>New &amp;Tag</string>
   </property>
  </action>
  <action name="actionNew_Branch" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/main/view_tree.png</normaloff>:/main/view_tree.png</iconset>
   </property>
   <property name="text" >
    <string>New &amp;Branch</string>
   </property>
  </action>
  <action name="action_Delete_Branch" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/main/button_cancel.png</normaloff>:/main/button_cancel.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Delete Branch</string>
   </property>
  </action>
  <action name="action_Remote_branches" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/main/view_sidetree.png</normaloff>:/main/view_sidetree.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Remote Branches</string>
   </property>
  </action>
  <action name="actionPush" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/main/top.png</normaloff>:/main/top.png</iconset>
   </property>
   <property name="text" >
    <string>Push</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc" />
 </resources>
 <connections/>
</ui>
